ruby_on_rails/sentry.md

```diff
@@ -1,17 +1,23 @@
 # Sentry
 
-## Setup Monitoring Services
+## General configuration
 
-* Go to https://www.sentry.io and login as the renuo monitor.
+* Go to https://www.sentry.io and login as the renuo monitor user.
 
-* Create a project for each environment (`master`, `develop`, `testing`). So: `[project-name]-master`, `[project-name]-develop`, `[project-name]-testing`
+* Create a project named `[project-name]`.
 
-* Add the `master` to the *#renuo* team, `testing` and `develop` to the *#no-notification* team. We are interested to receive notifications only regarding the `master` environment, but to log errors also in other environments.
+* Add the project to the *#renuo* team if the client pays for monitoring, to the `#no-notifications` otherwise.
 
-* Once you have created an entry, you will see the `dsn key` which you'll need in your config variables on Heroku. Note it.
+* Note the DSN key.
 
   ![sentry_dsn](../images/sentry.png)
 
+* Add the `SENTRY_DSN` value to the password manager.
```

Comment:
is it necessary? i always look it up on heroku or in sentry directly

Reply:
I had in mind of putting all the secrets you need for the `application.yml` in password manager, so you could find everything in one place instead of looking into different places.

Reply:
I remove it and we can discuss it separately